ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -11,6 +11,14 @@ base_images:
     name: "4.17"
     namespace: ocp
     tag: windows-machine-config-operator-bundle
+  previous-eus-bundle:
```

Comment:
Why only 4.16 -> 4.18?
Don't we need to be testing 4.12 -> 4.14 and  4.16 -> 4,18 as well?

Reply:
good question; continue in [slack](https://redhat-internal.slack.com/archives/CM4ERHBJS/p1725979869281719?thread_ts=1725885396.355509&cid=CM4ERHBJS)